src/coreclr/debug/daccess/dacdbiimpl.cpp

```diff
@@ -1256,32 +1250,9 @@ bool DacDbiInterfaceImpl::GetILImageInfoFromNgenPEFile(VMPTR_PEFile vmPEFile,
                                                        DWORD &dwSize,
                                                        IStringHolder* pStrFilename)
 {
-#if !defined(FEATURE_PREJIT)
 
     return false;
```

Comment:
This method is now a dead code. Should it be deleted?

Reply:
Yes. I wanted to keep this PR to just deleting things under the `FEATURE_PREJIT` define though, so I didn't do additional cleanup. 

There's definitely a bunch of methods that now always return true/false/null or do nothing such that they and their usage can be removed. I'd rather handle those separately from the initial coarse delete of the ifdef-ed code.